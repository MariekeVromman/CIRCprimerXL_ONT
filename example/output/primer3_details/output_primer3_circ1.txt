SEQUENCE_ID=circ1_chr1_17232_17742
SEQUENCE_TEMPLATE=AGGTTCTCGGTGGTGTTGAAGAGCAGCAAGGAGCTGACAGAGCTGATGTTGCTGGGAAGACCCCCAAGTCCCTCTTCTGCATCGTCCTCGGGCTCCGGCTTGGTGCTCACGCACACAGGAAAGTCCTTCAGCTTCTCCTGCTGTTCCAGCTGCTCTCTCTTGCTGATGGACAAGGGGGCATCAAACAGCTTCTCCTCTGTCTCTGCCCCCAGCATCACATGGGTCTTTGTTACAGCACCAGCCAGGGGGTCCAGGAAGACATACTTCTTCTAC
SEQUENCE_TARGET=135,1
PRIMER_NUM_RETURN=20
PRIMER_MIN_LEFT_THREE_PRIME_DISTANCE=1
PRIMER_PRODUCT_SIZE_RANGE=50-166
PRIMER_MIN_TM=58
PRIMER_MAX_TM=60
PRIMER_OPT_TM=59
PRIMER_PAIR_MAX_DIFF_TM=2
PRIMER_MIN_GC=30
PRIMER_MAX_GC=80
PRIMER_OPT_GC_PERCENT=50
SEQUENCE_EXCLUDED_REGION=23,4 32,4 102,4 110,4 173,7 203,7 233,1 235,1 237,2 244,2 247,1 249,1
PRIMER_LEFT_EXPLAIN=considered 196, overlap excluded region 61, GC content failed 3, low tm 32, high tm 36, high hairpin stability 1, long poly-x seq 8, ok 11
PRIMER_RIGHT_EXPLAIN=considered 242, overlap excluded region 91, GC content failed 1, low tm 63, high tm 50, ok 14
PRIMER_PAIR_EXPLAIN=considered 84, unacceptable product size 76, primer in pair overlaps a primer in a better pair 70, ok 8
PRIMER_LEFT_NUM_RETURNED=8
PRIMER_RIGHT_NUM_RETURNED=8
PRIMER_INTERNAL_NUM_RETURNED=0
PRIMER_PAIR_NUM_RETURNED=8
PRIMER_PAIR_0_PENALTY=2.115304
PRIMER_LEFT_0_PENALTY=1.531127
PRIMER_RIGHT_0_PENALTY=0.584177
PRIMER_LEFT_0_SEQUENCE=CTGATGTTGCTGGGAAGAC
PRIMER_RIGHT_0_SEQUENCE=GACAGAGGAGAAGCTGTTTG
PRIMER_LEFT_0=42,19
PRIMER_RIGHT_0=200,20
PRIMER_LEFT_0_TM=59.531
PRIMER_RIGHT_0_TM=59.584
PRIMER_LEFT_0_GC_PERCENT=52.632
PRIMER_RIGHT_0_GC_PERCENT=50.000
PRIMER_LEFT_0_SELF_ANY_TH=0.00
PRIMER_RIGHT_0_SELF_ANY_TH=0.00
PRIMER_LEFT_0_SELF_END_TH=0.00
PRIMER_RIGHT_0_SELF_END_TH=0.00
PRIMER_LEFT_0_HAIRPIN_TH=0.00
PRIMER_RIGHT_0_HAIRPIN_TH=45.79
PRIMER_LEFT_0_END_STABILITY=3.0100
PRIMER_RIGHT_0_END_STABILITY=2.9300
PRIMER_PAIR_0_COMPL_ANY_TH=0.00
PRIMER_PAIR_0_COMPL_END_TH=0.00
PRIMER_PAIR_0_PRODUCT_SIZE=159
PRIMER_PAIR_1_PENALTY=3.882131
PRIMER_LEFT_1_PENALTY=1.762244
PRIMER_RIGHT_1_PENALTY=2.119887
PRIMER_LEFT_1_SEQUENCE=GTTCTCGGTGGTGTTGAAG
PRIMER_RIGHT_1_SEQUENCE=AAGAGAGAGCAGCTGGAA
PRIMER_LEFT_1=2,19
PRIMER_RIGHT_1=160,18
PRIMER_LEFT_1_TM=59.762
PRIMER_RIGHT_1_TM=58.880
PRIMER_LEFT_1_GC_PERCENT=52.632
PRIMER_RIGHT_1_GC_PERCENT=50.000
PRIMER_LEFT_1_SELF_ANY_TH=0.00
PRIMER_RIGHT_1_SELF_ANY_TH=22.34
PRIMER_LEFT_1_SELF_END_TH=0.00
PRIMER_RIGHT_1_SELF_END_TH=0.00
PRIMER_LEFT_1_HAIRPIN_TH=0.00
PRIMER_RIGHT_1_HAIRPIN_TH=0.00
PRIMER_LEFT_1_END_STABILITY=3.0200
PRIMER_RIGHT_1_END_STABILITY=3.5300
PRIMER_PAIR_1_COMPL_ANY_TH=0.00
PRIMER_PAIR_1_COMPL_END_TH=0.00
PRIMER_PAIR_1_PRODUCT_SIZE=159
PRIMER_PAIR_2_PENALTY=4.021910
PRIMER_LEFT_2_PENALTY=1.902023
PRIMER_RIGHT_2_PENALTY=2.119887
PRIMER_LEFT_2_SEQUENCE=TTCTCGGTGGTGTTGAAGA
PRIMER_RIGHT_2_SEQUENCE=AAGAGAGAGCAGCTGGAA
PRIMER_LEFT_2=3,19
PRIMER_RIGHT_2=160,18
PRIMER_LEFT_2_TM=59.902
PRIMER_RIGHT_2_TM=58.880
PRIMER_LEFT_2_GC_PERCENT=47.368
PRIMER_RIGHT_2_GC_PERCENT=50.000
PRIMER_LEFT_2_SELF_ANY_TH=0.00
PRIMER_RIGHT_2_SELF_ANY_TH=22.34
PRIMER_LEFT_2_SELF_END_TH=0.00
PRIMER_RIGHT_2_SELF_END_TH=0.00
PRIMER_LEFT_2_HAIRPIN_TH=0.00
PRIMER_RIGHT_2_HAIRPIN_TH=0.00
PRIMER_LEFT_2_END_STABILITY=2.8700
PRIMER_RIGHT_2_END_STABILITY=3.5300
PRIMER_PAIR_2_COMPL_ANY_TH=0.00
PRIMER_PAIR_2_COMPL_END_TH=0.00
PRIMER_PAIR_2_PRODUCT_SIZE=158
PRIMER_PAIR_3_PENALTY=4.139896
PRIMER_LEFT_3_PENALTY=3.555719
PRIMER_RIGHT_3_PENALTY=0.584177
PRIMER_LEFT_3_SEQUENCE=TCCCTCTTCTGCATCGT
PRIMER_RIGHT_3_SEQUENCE=GACAGAGGAGAAGCTGTTTG
PRIMER_LEFT_3=68,17
PRIMER_RIGHT_3=200,20
PRIMER_LEFT_3_TM=58.444
PRIMER_RIGHT_3_TM=59.584
PRIMER_LEFT_3_GC_PERCENT=52.941
PRIMER_RIGHT_3_GC_PERCENT=50.000
PRIMER_LEFT_3_SELF_ANY_TH=0.00
PRIMER_RIGHT_3_SELF_ANY_TH=0.00
PRIMER_LEFT_3_SELF_END_TH=0.00
PRIMER_RIGHT_3_SELF_END_TH=0.00
PRIMER_LEFT_3_HAIRPIN_TH=0.00
PRIMER_RIGHT_3_HAIRPIN_TH=45.79
PRIMER_LEFT_3_END_STABILITY=3.7300
PRIMER_RIGHT_3_END_STABILITY=2.9300
PRIMER_PAIR_3_COMPL_ANY_TH=13.83
PRIMER_PAIR_3_COMPL_END_TH=1.91
PRIMER_PAIR_3_PRODUCT_SIZE=133
PRIMER_PAIR_4_PENALTY=4.423729
PRIMER_LEFT_4_PENALTY=2.303841
PRIMER_RIGHT_4_PENALTY=2.119887
PRIMER_LEFT_4_SEQUENCE=CTCGGTGGTGTTGAAGAG
PRIMER_RIGHT_4_SEQUENCE=AAGAGAGAGCAGCTGGAA
PRIMER_LEFT_4=5,18
PRIMER_RIGHT_4=160,18
PRIMER_LEFT_4_TM=58.696
PRIMER_RIGHT_4_TM=58.880
PRIMER_LEFT_4_GC_PERCENT=55.556
PRIMER_RIGHT_4_GC_PERCENT=50.000
PRIMER_LEFT_4_SELF_ANY_TH=0.00
PRIMER_RIGHT_4_SELF_ANY_TH=22.34
PRIMER_LEFT_4_SELF_END_TH=0.00
PRIMER_RIGHT_4_SELF_END_TH=0.00
PRIMER_LEFT_4_HAIRPIN_TH=0.00
PRIMER_RIGHT_4_HAIRPIN_TH=0.00
PRIMER_LEFT_4_END_STABILITY=2.8500
PRIMER_RIGHT_4_END_STABILITY=3.5300
PRIMER_PAIR_4_COMPL_ANY_TH=0.00
PRIMER_PAIR_4_COMPL_END_TH=0.00
PRIMER_PAIR_4_PRODUCT_SIZE=156
PRIMER_PAIR_5_PENALTY=4.597563
PRIMER_LEFT_5_PENALTY=2.477676
PRIMER_RIGHT_5_PENALTY=2.119887
PRIMER_LEFT_5_SEQUENCE=GTTCTCGGTGGTGTTGAA
PRIMER_RIGHT_5_SEQUENCE=AAGAGAGAGCAGCTGGAA
PRIMER_LEFT_5=2,18
PRIMER_RIGHT_5=160,18
PRIMER_LEFT_5_TM=58.522
PRIMER_RIGHT_5_TM=58.880
PRIMER_LEFT_5_GC_PERCENT=50.000
PRIMER_RIGHT_5_GC_PERCENT=50.000
PRIMER_LEFT_5_SELF_ANY_TH=0.00
PRIMER_RIGHT_5_SELF_ANY_TH=22.34
PRIMER_LEFT_5_SELF_END_TH=0.00
PRIMER_RIGHT_5_SELF_END_TH=0.00
PRIMER_LEFT_5_HAIRPIN_TH=0.00
PRIMER_RIGHT_5_HAIRPIN_TH=0.00
PRIMER_LEFT_5_END_STABILITY=2.6900
PRIMER_RIGHT_5_END_STABILITY=3.5300
PRIMER_PAIR_5_COMPL_ANY_TH=0.00
PRIMER_PAIR_5_COMPL_END_TH=0.00
PRIMER_PAIR_5_PRODUCT_SIZE=159
PRIMER_PAIR_6_PENALTY=5.162353
PRIMER_LEFT_6_PENALTY=3.042466
PRIMER_RIGHT_6_PENALTY=2.119887
PRIMER_LEFT_6_SEQUENCE=AGGTTCTCGGTGGTGTT
PRIMER_RIGHT_6_SEQUENCE=AAGAGAGAGCAGCTGGAA
PRIMER_LEFT_6=0,17
PRIMER_RIGHT_6=160,18
PRIMER_LEFT_6_TM=58.958
PRIMER_RIGHT_6_TM=58.880
PRIMER_LEFT_6_GC_PERCENT=52.941
PRIMER_RIGHT_6_GC_PERCENT=50.000
PRIMER_LEFT_6_SELF_ANY_TH=0.00
PRIMER_RIGHT_6_SELF_ANY_TH=22.34
PRIMER_LEFT_6_SELF_END_TH=0.00
PRIMER_RIGHT_6_SELF_END_TH=0.00
PRIMER_LEFT_6_HAIRPIN_TH=0.00
PRIMER_RIGHT_6_HAIRPIN_TH=0.00
PRIMER_LEFT_6_END_STABILITY=3.3200
PRIMER_RIGHT_6_END_STABILITY=3.5300
PRIMER_PAIR_6_COMPL_ANY_TH=0.00
PRIMER_PAIR_6_COMPL_END_TH=0.00
PRIMER_PAIR_6_PRODUCT_SIZE=161
PRIMER_PAIR_7_PENALTY=5.280892
PRIMER_LEFT_7_PENALTY=3.161004
PRIMER_RIGHT_7_PENALTY=2.119887
PRIMER_LEFT_7_SEQUENCE=GGTTCTCGGTGGTGTTG
PRIMER_RIGHT_7_SEQUENCE=AAGAGAGAGCAGCTGGAA
PRIMER_LEFT_7=1,17
PRIMER_RIGHT_7=160,18
PRIMER_LEFT_7_TM=58.839
PRIMER_RIGHT_7_TM=58.880
PRIMER_LEFT_7_GC_PERCENT=58.824
PRIMER_RIGHT_7_GC_PERCENT=50.000
PRIMER_LEFT_7_SELF_ANY_TH=0.00
PRIMER_RIGHT_7_SELF_ANY_TH=22.34
PRIMER_LEFT_7_SELF_END_TH=0.00
PRIMER_RIGHT_7_SELF_END_TH=0.00
PRIMER_LEFT_7_HAIRPIN_TH=0.00
PRIMER_RIGHT_7_HAIRPIN_TH=0.00
PRIMER_LEFT_7_END_STABILITY=3.3300
PRIMER_RIGHT_7_END_STABILITY=3.5300
PRIMER_PAIR_7_COMPL_ANY_TH=0.00
PRIMER_PAIR_7_COMPL_END_TH=0.00
PRIMER_PAIR_7_PRODUCT_SIZE=160
=
