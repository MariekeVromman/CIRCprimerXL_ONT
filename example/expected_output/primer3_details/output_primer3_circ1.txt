SEQUENCE_ID=circ1_chr1_17232_17742
SEQUENCE_TEMPLATE=CCCTCTGGCCATACAGGTTCTCGGTGGTGTTGAAGAGCAGCAAGGAGCTGACAGAGCTGATGTTGCTGGGAAGACCCCCAAGTCCCTCTTCTGCATCGTCCTCGGGCTCCGGCTTGGTGCTCACGCACACAGGAAAGTCCTTCAGCTTCTCCTGCTGTTCCAGCTGCTCTCTCTTGCTGATGGACAAGGGGGCATCAAACAGCTTCTCCTCTGTCTCTGCCCCCAGCATCACATGGGTCTTTGTTACAGCACCAGCCAGGGGGTCCAGGAAGACATACTTCTTCTACCTACAGAGGCGACAT
SEQUENCE_TARGET=149,1
PRIMER_NUM_RETURN=20
PRIMER_MIN_LEFT_THREE_PRIME_DISTANCE=1
SEQUENCE_EXCLUDED_REGION=37,4 46,4 116,4 124,4 187,7 217,7 247,1 249,1 251,2 258,2 261,1 263,1
PRIMER_LEFT_EXPLAIN=considered 294, overlap excluded region 61, GC content failed 3, low tm 39, high tm 112, high hairpin stability 1, long poly-x seq 8, ok 16
PRIMER_RIGHT_EXPLAIN=considered 418, overlap excluded region 91, GC content failed 1, low tm 143, high tm 125, ok 33
PRIMER_PAIR_EXPLAIN=considered 12, unacceptable product size 1, primer in pair overlaps a primer in a better pair 517, ok 11
PRIMER_LEFT_NUM_RETURNED=11
PRIMER_RIGHT_NUM_RETURNED=11
PRIMER_INTERNAL_NUM_RETURNED=0
PRIMER_PAIR_NUM_RETURNED=11
PRIMER_PAIR_0_PENALTY=2.115304
PRIMER_LEFT_0_PENALTY=1.531127
PRIMER_RIGHT_0_PENALTY=0.584177
PRIMER_LEFT_0_SEQUENCE=CTGATGTTGCTGGGAAGAC
PRIMER_RIGHT_0_SEQUENCE=GACAGAGGAGAAGCTGTTTG
PRIMER_LEFT_0=56,19
PRIMER_RIGHT_0=214,20
PRIMER_LEFT_0_TM=59.531
PRIMER_RIGHT_0_TM=59.584
PRIMER_LEFT_0_GC_PERCENT=52.632
PRIMER_RIGHT_0_GC_PERCENT=50.000
PRIMER_LEFT_0_SELF_ANY_TH=0.00
PRIMER_RIGHT_0_SELF_ANY_TH=0.00
PRIMER_LEFT_0_SELF_END_TH=0.00
PRIMER_RIGHT_0_SELF_END_TH=0.00
PRIMER_LEFT_0_HAIRPIN_TH=0.00
PRIMER_RIGHT_0_HAIRPIN_TH=45.79
PRIMER_LEFT_0_END_STABILITY=3.0100
PRIMER_RIGHT_0_END_STABILITY=2.9300
PRIMER_PAIR_0_COMPL_ANY_TH=0.00
PRIMER_PAIR_0_COMPL_END_TH=0.00
PRIMER_PAIR_0_PRODUCT_SIZE=159
PRIMER_PAIR_1_PENALTY=2.346421
PRIMER_LEFT_1_PENALTY=1.762244
PRIMER_RIGHT_1_PENALTY=0.584177
PRIMER_LEFT_1_SEQUENCE=GTTCTCGGTGGTGTTGAAG
PRIMER_RIGHT_1_SEQUENCE=GACAGAGGAGAAGCTGTTTG
PRIMER_LEFT_1=16,19
PRIMER_RIGHT_1=214,20
PRIMER_LEFT_1_TM=59.762
PRIMER_RIGHT_1_TM=59.584
PRIMER_LEFT_1_GC_PERCENT=52.632
PRIMER_RIGHT_1_GC_PERCENT=50.000
PRIMER_LEFT_1_SELF_ANY_TH=0.00
PRIMER_RIGHT_1_SELF_ANY_TH=0.00
PRIMER_LEFT_1_SELF_END_TH=0.00
PRIMER_RIGHT_1_SELF_END_TH=0.00
PRIMER_LEFT_1_HAIRPIN_TH=0.00
PRIMER_RIGHT_1_HAIRPIN_TH=45.79
PRIMER_LEFT_1_END_STABILITY=3.0200
PRIMER_RIGHT_1_END_STABILITY=2.9300
PRIMER_PAIR_1_COMPL_ANY_TH=0.00
PRIMER_PAIR_1_COMPL_END_TH=0.00
PRIMER_PAIR_1_PRODUCT_SIZE=199
PRIMER_PAIR_2_PENALTY=2.406625
PRIMER_LEFT_2_PENALTY=1.822448
PRIMER_RIGHT_2_PENALTY=0.584177
PRIMER_LEFT_2_SEQUENCE=CTCTGGCCATACAGGTTCT
PRIMER_RIGHT_2_SEQUENCE=GACAGAGGAGAAGCTGTTTG
PRIMER_LEFT_2=2,19
PRIMER_RIGHT_2=214,20
PRIMER_LEFT_2_TM=59.822
PRIMER_RIGHT_2_TM=59.584
PRIMER_LEFT_2_GC_PERCENT=52.632
PRIMER_RIGHT_2_GC_PERCENT=50.000
PRIMER_LEFT_2_SELF_ANY_TH=12.32
PRIMER_RIGHT_2_SELF_ANY_TH=0.00
PRIMER_LEFT_2_SELF_END_TH=0.00
PRIMER_RIGHT_2_SELF_END_TH=0.00
PRIMER_LEFT_2_HAIRPIN_TH=41.12
PRIMER_RIGHT_2_HAIRPIN_TH=45.79
PRIMER_LEFT_2_END_STABILITY=3.0100
PRIMER_RIGHT_2_END_STABILITY=2.9300
PRIMER_PAIR_2_COMPL_ANY_TH=22.47
PRIMER_PAIR_2_COMPL_END_TH=22.47
PRIMER_PAIR_2_PRODUCT_SIZE=213
PRIMER_PAIR_3_PENALTY=2.407627
PRIMER_LEFT_3_PENALTY=1.823450
PRIMER_RIGHT_3_PENALTY=0.584177
PRIMER_LEFT_3_SEQUENCE=TCTGGCCATACAGGTTCTC
PRIMER_RIGHT_3_SEQUENCE=GACAGAGGAGAAGCTGTTTG
PRIMER_LEFT_3=3,19
PRIMER_RIGHT_3=214,20
PRIMER_LEFT_3_TM=59.823
PRIMER_RIGHT_3_TM=59.584
PRIMER_LEFT_3_GC_PERCENT=52.632
PRIMER_RIGHT_3_GC_PERCENT=50.000
PRIMER_LEFT_3_SELF_ANY_TH=12.32
PRIMER_RIGHT_3_SELF_ANY_TH=0.00
PRIMER_LEFT_3_SELF_END_TH=0.00
PRIMER_RIGHT_3_SELF_END_TH=0.00
PRIMER_LEFT_3_HAIRPIN_TH=41.12
PRIMER_RIGHT_3_HAIRPIN_TH=45.79
PRIMER_LEFT_3_END_STABILITY=2.8700
PRIMER_RIGHT_3_END_STABILITY=2.9300
PRIMER_PAIR_3_COMPL_ANY_TH=29.59
PRIMER_PAIR_3_COMPL_END_TH=29.59
PRIMER_PAIR_3_PRODUCT_SIZE=212
PRIMER_PAIR_4_PENALTY=2.486200
PRIMER_LEFT_4_PENALTY=1.902023
PRIMER_RIGHT_4_PENALTY=0.584177
PRIMER_LEFT_4_SEQUENCE=TTCTCGGTGGTGTTGAAGA
PRIMER_RIGHT_4_SEQUENCE=GACAGAGGAGAAGCTGTTTG
PRIMER_LEFT_4=17,19
PRIMER_RIGHT_4=214,20
PRIMER_LEFT_4_TM=59.902
PRIMER_RIGHT_4_TM=59.584
PRIMER_LEFT_4_GC_PERCENT=47.368
PRIMER_RIGHT_4_GC_PERCENT=50.000
PRIMER_LEFT_4_SELF_ANY_TH=0.00
PRIMER_RIGHT_4_SELF_ANY_TH=0.00
PRIMER_LEFT_4_SELF_END_TH=0.00
PRIMER_RIGHT_4_SELF_END_TH=0.00
PRIMER_LEFT_4_HAIRPIN_TH=0.00
PRIMER_RIGHT_4_HAIRPIN_TH=45.79
PRIMER_LEFT_4_END_STABILITY=2.8700
PRIMER_RIGHT_4_END_STABILITY=2.9300
PRIMER_PAIR_4_COMPL_ANY_TH=0.00
PRIMER_PAIR_4_COMPL_END_TH=0.00
PRIMER_PAIR_4_PRODUCT_SIZE=198
PRIMER_PAIR_5_PENALTY=2.888018
PRIMER_LEFT_5_PENALTY=2.303841
PRIMER_RIGHT_5_PENALTY=0.584177
PRIMER_LEFT_5_SEQUENCE=CTCGGTGGTGTTGAAGAG
PRIMER_RIGHT_5_SEQUENCE=GACAGAGGAGAAGCTGTTTG
PRIMER_LEFT_5=19,18
PRIMER_RIGHT_5=214,20
PRIMER_LEFT_5_TM=58.696
PRIMER_RIGHT_5_TM=59.584
PRIMER_LEFT_5_GC_PERCENT=55.556
PRIMER_RIGHT_5_GC_PERCENT=50.000
PRIMER_LEFT_5_SELF_ANY_TH=0.00
PRIMER_RIGHT_5_SELF_ANY_TH=0.00
PRIMER_LEFT_5_SELF_END_TH=0.00
PRIMER_RIGHT_5_SELF_END_TH=0.00
PRIMER_LEFT_5_HAIRPIN_TH=0.00
PRIMER_RIGHT_5_HAIRPIN_TH=45.79
PRIMER_LEFT_5_END_STABILITY=2.8500
PRIMER_RIGHT_5_END_STABILITY=2.9300
PRIMER_PAIR_5_COMPL_ANY_TH=0.00
PRIMER_PAIR_5_COMPL_END_TH=0.00
PRIMER_PAIR_5_PRODUCT_SIZE=196
PRIMER_PAIR_6_PENALTY=3.061853
PRIMER_LEFT_6_PENALTY=2.477676
PRIMER_RIGHT_6_PENALTY=0.584177
PRIMER_LEFT_6_SEQUENCE=GTTCTCGGTGGTGTTGAA
PRIMER_RIGHT_6_SEQUENCE=GACAGAGGAGAAGCTGTTTG
PRIMER_LEFT_6=16,18
PRIMER_RIGHT_6=214,20
PRIMER_LEFT_6_TM=58.522
PRIMER_RIGHT_6_TM=59.584
PRIMER_LEFT_6_GC_PERCENT=50.000
PRIMER_RIGHT_6_GC_PERCENT=50.000
PRIMER_LEFT_6_SELF_ANY_TH=0.00
PRIMER_RIGHT_6_SELF_ANY_TH=0.00
PRIMER_LEFT_6_SELF_END_TH=0.00
PRIMER_RIGHT_6_SELF_END_TH=0.00
PRIMER_LEFT_6_HAIRPIN_TH=0.00
PRIMER_RIGHT_6_HAIRPIN_TH=45.79
PRIMER_LEFT_6_END_STABILITY=2.6900
PRIMER_RIGHT_6_END_STABILITY=2.9300
PRIMER_PAIR_6_COMPL_ANY_TH=0.00
PRIMER_PAIR_6_COMPL_END_TH=0.00
PRIMER_PAIR_6_PRODUCT_SIZE=199
PRIMER_PAIR_7_PENALTY=3.281218
PRIMER_LEFT_7_PENALTY=2.697041
PRIMER_RIGHT_7_PENALTY=0.584177
PRIMER_LEFT_7_SEQUENCE=CCTCTGGCCATACAGGTT
PRIMER_RIGHT_7_SEQUENCE=GACAGAGGAGAAGCTGTTTG
PRIMER_LEFT_7=1,18
PRIMER_RIGHT_7=214,20
PRIMER_LEFT_7_TM=59.697
PRIMER_RIGHT_7_TM=59.584
PRIMER_LEFT_7_GC_PERCENT=55.556
PRIMER_RIGHT_7_GC_PERCENT=50.000
PRIMER_LEFT_7_SELF_ANY_TH=12.32
PRIMER_RIGHT_7_SELF_ANY_TH=0.00
PRIMER_LEFT_7_SELF_END_TH=0.00
PRIMER_RIGHT_7_SELF_END_TH=0.00
PRIMER_LEFT_7_HAIRPIN_TH=41.12
PRIMER_RIGHT_7_HAIRPIN_TH=45.79
PRIMER_LEFT_7_END_STABILITY=3.5000
PRIMER_RIGHT_7_END_STABILITY=2.9300
PRIMER_PAIR_7_COMPL_ANY_TH=17.11
PRIMER_PAIR_7_COMPL_END_TH=12.91
PRIMER_PAIR_7_PRODUCT_SIZE=214
PRIMER_PAIR_8_PENALTY=3.626643
PRIMER_LEFT_8_PENALTY=3.042466
PRIMER_RIGHT_8_PENALTY=0.584177
PRIMER_LEFT_8_SEQUENCE=AGGTTCTCGGTGGTGTT
PRIMER_RIGHT_8_SEQUENCE=GACAGAGGAGAAGCTGTTTG
PRIMER_LEFT_8=14,17
PRIMER_RIGHT_8=214,20
PRIMER_LEFT_8_TM=58.958
PRIMER_RIGHT_8_TM=59.584
PRIMER_LEFT_8_GC_PERCENT=52.941
PRIMER_RIGHT_8_GC_PERCENT=50.000
PRIMER_LEFT_8_SELF_ANY_TH=0.00
PRIMER_RIGHT_8_SELF_ANY_TH=0.00
PRIMER_LEFT_8_SELF_END_TH=0.00
PRIMER_RIGHT_8_SELF_END_TH=0.00
PRIMER_LEFT_8_HAIRPIN_TH=0.00
PRIMER_RIGHT_8_HAIRPIN_TH=45.79
PRIMER_LEFT_8_END_STABILITY=3.3200
PRIMER_RIGHT_8_END_STABILITY=2.9300
PRIMER_PAIR_8_COMPL_ANY_TH=11.90
PRIMER_PAIR_8_COMPL_END_TH=0.00
PRIMER_PAIR_8_PRODUCT_SIZE=201
PRIMER_PAIR_9_PENALTY=3.745181
PRIMER_LEFT_9_PENALTY=3.161004
PRIMER_RIGHT_9_PENALTY=0.584177
PRIMER_LEFT_9_SEQUENCE=GGTTCTCGGTGGTGTTG
PRIMER_RIGHT_9_SEQUENCE=GACAGAGGAGAAGCTGTTTG
PRIMER_LEFT_9=15,17
PRIMER_RIGHT_9=214,20
PRIMER_LEFT_9_TM=58.839
PRIMER_RIGHT_9_TM=59.584
PRIMER_LEFT_9_GC_PERCENT=58.824
PRIMER_RIGHT_9_GC_PERCENT=50.000
PRIMER_LEFT_9_SELF_ANY_TH=0.00
PRIMER_RIGHT_9_SELF_ANY_TH=0.00
PRIMER_LEFT_9_SELF_END_TH=0.00
PRIMER_RIGHT_9_SELF_END_TH=0.00
PRIMER_LEFT_9_HAIRPIN_TH=0.00
PRIMER_RIGHT_9_HAIRPIN_TH=45.79
PRIMER_LEFT_9_END_STABILITY=3.3300
PRIMER_RIGHT_9_END_STABILITY=2.9300
PRIMER_PAIR_9_COMPL_ANY_TH=6.31
PRIMER_PAIR_9_COMPL_END_TH=0.00
PRIMER_PAIR_9_PRODUCT_SIZE=200
PRIMER_PAIR_10_PENALTY=4.139896
PRIMER_LEFT_10_PENALTY=3.555719
PRIMER_RIGHT_10_PENALTY=0.584177
PRIMER_LEFT_10_SEQUENCE=TCCCTCTTCTGCATCGT
PRIMER_RIGHT_10_SEQUENCE=GACAGAGGAGAAGCTGTTTG
PRIMER_LEFT_10=82,17
PRIMER_RIGHT_10=214,20
PRIMER_LEFT_10_TM=58.444
PRIMER_RIGHT_10_TM=59.584
PRIMER_LEFT_10_GC_PERCENT=52.941
PRIMER_RIGHT_10_GC_PERCENT=50.000
PRIMER_LEFT_10_SELF_ANY_TH=0.00
PRIMER_RIGHT_10_SELF_ANY_TH=0.00
PRIMER_LEFT_10_SELF_END_TH=0.00
PRIMER_RIGHT_10_SELF_END_TH=0.00
PRIMER_LEFT_10_HAIRPIN_TH=0.00
PRIMER_RIGHT_10_HAIRPIN_TH=45.79
PRIMER_LEFT_10_END_STABILITY=3.7300
PRIMER_RIGHT_10_END_STABILITY=2.9300
PRIMER_PAIR_10_COMPL_ANY_TH=13.83
PRIMER_PAIR_10_COMPL_END_TH=1.91
PRIMER_PAIR_10_PRODUCT_SIZE=133
=
